### PR TITLE
Expand nature of rate limit changes in 1.43

### PR DESCRIPTION
**Pull request by sbesson**
Document breaking API changes as well as new API additions

### CODE REVIEW

**Comment by sfdye** [comment]
Probably worth keep the `rate`` attribute, since Github response still has it. I will issue a deprecation warning if the user tries to access the `rate` instead of the `core`. How's that sound?

**Comment by sfdye** [comment]
Release [1.43.2](https://github.com/PyGithub/PyGithub/releases/tag/v1.43.2)

This should fix the problem.

**Comment by sfdye** [comment]
So can we close this?

File: doc/changes.rst [hunk 1]
```diff
@@ -27,6 +27,9 @@ Version 1.43 (September 08, 2018)
 * Add support for required approving review count (#888) (ef16702)
 * Add ``Organization.invite_user`` (880)(eb80564)
 * Add support for search/graphql rate limit (fd8a036)
+
+  + (**breaking**) Remove ``RateLimit.rate``
```
Comment (sfdye) [inline doc/changes.rst]: Maybe you can change this deprecation instead？

**Comment by sfdye** [comment]
Agreed. A side note will help. I will merge once you make the change!

**Comment by sbesson** [comment]
@sfdye pushed

**Pull request merged**